src/libs/actions/ReimbursementAccount/deleteFromBankAccountList.js

```diff
@@ -0,0 +1,18 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../../ONYXKEYS';
+import * as store from './store';
+
+/**
+ * Deletes a bank account from bankAccountList
+ *
+ * @param {Number} bankAccountID
+ */
+function deleteFromBankAccountList(bankAccountID) {
+    // We should delete the bankAccountID key from the bankAccountList object before setting it in Onyx
+    const bankAccountList = store.getBankAccountList();
+    delete bankAccountList[bankAccountID];
+
+    Onyx.merge(ONYXKEYS.BANK_ACCOUNT_LIST, bankAccountList);
+}
```

Comment:
Is there a reason for creating a new file for each of these functions/actions? I think it would only make sense if we had a unique implementation for each platform... Am I missing something?

Reply:
I tried to create `deleteFromBankAccountList` in both `ReimbursementAccount/index.js` and `BankAccounts.js` and I was getting a circular dependency error. Creating it in a separate file was one way I found around it, and presumably why the other functions were created in separate files too. I'm not sure why we import `ReimbursementAccount` and then export it as well, but maybe it's time for a refactor in a separate PR?

Reply:
Cool, lets stick with what you have for now then.

Reply:
Cool! Let me know if you have any other questions!